jobs/service-fabrik-broker/spec

```diff
@@ -27,6 +27,7 @@ provides:
   - password
   - skip_ssl_validation
   - log_level
+  - enable_bosh_rate_limit
```

Comment:
@abh1kg : can you please make a change to populate the default value for this configuration as false ?